packages/experimental/Checkbox/src/useCheckbox.ts

```diff
@@ -54,7 +54,7 @@ export const useCheckbox = (props: CheckboxProps): CheckboxInfo => {
     ...pressable.state,
     disabled: !!props.disabled,
     checked: isChecked,
-    boxAtEnd: boxSide == undefined || boxSide == 'start' ? false : true,
+    boxAtEnd: labelPosition === 'after' ? true : false,
```

Comment:
Intentional to not also change boxAtEnd? I guess that's a bigger change because a lot of settings refer to that.

Reply:
Yeah we can rename it, but since it's not something that's used by clients, we don't really need to rename it.

Reply:
I think I slightly prefer it because labelPosition is easier to understand than "boxAtEnd", but it's not review blocking

Reply:
Sure, done